### DISCUSSION TITLE
希望将v2中清爽首页的排行榜改回v1的10个

### DISCUSSION THREAD

**Discussion by crasher32** [the1812/Bilibili-Evolved#3175] [功能建议 / Ideas]
v2的清爽首页中，每个分区的排行榜默认只显示前三个，且不可修改，个人感觉并不太够
像v1版列出10个就很好，如果空间不够的话设置为滑动浏览也可以
![2022-03-24_20-19-09](https://user-images.githubusercontent.com/77274198/159914869-5fd8e9c1-3807-45cd-a081-33965d705b97.png)

**Comment by the1812** [comment]
点"排行榜"那行字就会打开完整的排行榜

**Reply by crasher32** [reply]
> 点"排行榜"那行字就会打开完整的排行榜

好的，谢谢！

**Comment by Ychan98** [comment]
说实话我个人也比较喜欢v1的样式，现在排行只有3个，体现不出什么作用，点击又凭空多一个步骤，另外v1的时间表看当日更新的番剧很方便，点击切换日期也不繁琐，现在的分别横向滚动好像个密码锁，
